src/types/EmbeddedPaymentElement.tsx

```diff
@@ -223,7 +261,32 @@ async function createEmbeddedPaymentElement(
   return new EmbeddedPaymentElement();
 }
 
-function setupConfirmHandlers(
+function rowSelectionConfigurationIsValid(
+  configuration: EmbeddedPaymentElementConfiguration
+): boolean {
+  // Confgiruation is invalid if:
+  // 1. Row selection behavior is immediateAction AND
+  // 2. Form sheet action is confirm AND
+  // 3. Either Apple Pay is enabled OR customer configuration is present
+
+  const isImmediateAction =
+    configuration.rowSelectionBehavior?.type === 'immediateAction';
+  const isFormSheetConfirm = configuration.formSheetAction?.type === 'confirm';
+  const isApplePayEnabled = !!configuration.applePay;
+  const hasCustomerConfig = !!configuration.customerId;
```

Comment:
Can we not rely on the native code to error out here? I guess Android and iOS error at different times, but seems possible?